.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Enhancement Request
+about: Suggest an enhancement to the OpenPAI project
+
+---
+<!-- Please only use this template for submitting enhancement requests -->
+
+**What would you like to be added**:
+
+**Why is this needed**:
+
```

Comment:
How it works without this feature currently? (workaround)

Reply:
added Without this feature, how does the current module work?